components/application-broker/internal/broker/provision.go

```diff
@@ -394,6 +422,53 @@ func (svc *ProvisionService) channelForApp(applicationName internal.ApplicationN
 	return svc.knClient.GetChannelByLabels(integrationNamespace, labels)
 }
 
+// Create a new policy
+func (svc *ProvisionService) createIstioPolicy(ns internal.Namespace) error {
+	policyName := string(ns) + policyNameSuffix
```

Comment:
If we can make a namespace variable that is of string type then we can avoid casting twice.
1: line 427
2: line 448